documentation/docs/02-template-syntax/01-svelte-components.md

```diff
@@ -321,7 +321,7 @@ To apply styles to a selector globally, use the `:global(...)` modifier.
 
 If you want to make @keyframes that are accessible globally, you need to prepend your keyframe names with `-global-`.
 
-The `-global-` part will be removed when compiled, and the keyframe then be referenced using just `my-animation-name` elsewhere in your code.
+The `-global-` part will be removed when compiled, and the keyframe will then be referenced using just `my-animation-name` elsewhere in your code.
```

Comment:
```suggestion
The `-global-` part will be removed when compiled, and the keyframe then be referenced using just `my-animation-name` elsewhere in your code.
```

I believe it's grammatical with or without the second "will", and I think it sounds less stilted without it.

Reply:
Hi @Conduitry,

It's technically correct in the same way that pirates can say "Arrr Billy, that boat be a fine piece of work!" but these days (unless you are a pirate at a party!) it sounds more like an oversight of someone who is not fully familiar with modern English usage or the language itself, for example this is a common error made by people who speak English as a second language.  I say this as someone who has a family full of them (I was born here in Canada, but half my family is from Estonia).

Given this is technical documentation, I think clarity and congruence with technical documentation norms avoid distractions from the topic due to unusual grammatical styling; the objective is to facilitate an efficient assimilation of the subject matter. "Stilted" versus "colloquial in a comical way" is in my opinion a better choice, but I'm not a moderator.

I will say it was jarring enough that I took the time to PR it, but again, I'm not a final decision-maker here.